### PR TITLE
0.13 docs: add cert manager docs dns and tls

### PR DESCRIPTION
**Pull request by ShankyJS**
- docs(k8s): ext dns and cert manager example (#3988)
- chore: Rework cert-manager-ext-dns example to Bonsai🪴

**What this PR does / why we need it**:

**Which issue(s) this PR fixes**:

Fixes #

**Special notes for your reviewer**:

### CODE REVIEW

File: examples/cert-manager-ext-dns/project.garden.yml [hunk 1]
```diff
@@ -0,0 +1,45 @@
...
+  DEFAULT_NAMESPACE: default # Frontend will be deployed in this namespace.
+  # Cert-Manager variables
+  CERT_MANAGER_INSTALL_CRDS: true
+  GENERATE_PROD_CERTS: false # Set this to true if you want to generate production certificates
+  GENERATE_STG_CERTS: false # Set this to true if you want to generate staging certificates
+  # External-DNS variables
+  CF_DNS_PROVIDER: cloudflare # This demo only supports Cloudflare at the moment
+  CF_DOMAIN: [ shankyjs.com ] # Has to be an array, even if you only have one domain
+  CF_UPDATE_STRATEGY: sync # Be careful with this, it can cause downtime if you're not careful
+  CF_EMAIL: jhan.silva@icloud.com # This is the email you use to login to Cloudflare
```
Comment (Orzelius) [inline examples/cert-manager-ext-dns/project.garden.yml:44]: You might start getting spam if you have it in the open here

**Review by stefreak — APPROVED** [review]
To me personally this looks good to go 👍

File: examples/cert-manager-ext-dns/project.garden.yml (hunk 1 above)
Comment (vvagaytsev) [reply examples/cert-manager-ext-dns/project.garden.yml:44]: @ShankyJS would you like to change this email address before we merge this to `0.13`?

Comment (ShankyJS) [reply examples/cert-manager-ext-dns/project.garden.yml]: Removed them @vvagaytsev! Thanks

**Pull request merged**